app/src/main/res/values/strings.xml

```diff
@@ -671,6 +671,7 @@
     <string name="recent">Recent</string>
     <string name="chapters">Chapters</string>
     <string name="no_app_to_open_intent">No app on your device can open this</string>
+    <string name="no_appropriate_file_manager_message">No appropriate file manager was found for this action.\nPlease install a file manager or try to enable/disable \'%s\' in the download settings.</string>
```

Comment:
@litetex have you seen this?

> Also, the string should be different for users on Android 10+, since there SAF is not disableable, so it is misleading to show a message that suggests to disable it.

Reply:
Added a switch for Android 10+ to use a different - better understandable - string.